db/schema.rb

```diff
@@ -12,38 +12,38 @@
 
 ActiveRecord::Schema[7.0].define(version: 2023_08_23_124431) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
+  enable_extension 'plpgsql'
```

Comment:
Did you manually update this file? 🤔

Reply:
Yes, I went over the file with the formatter and changed the quotation marks